src/actions/roundCorners/RoundCornersAction.ts

```diff
@@ -0,0 +1,43 @@
+import Action from "../Action";
+import {IAction} from "../../interfaces/IAction";
+import Param from "../../parameters/Param";
+import ParamValue from "../../parameters/ParamValue";
+
+/**
+ * @private
+ * @implements IAction
+ */
+class RoundCornersAction extends Action {
+  private param: Param;
+  constructor() {
+    super();
+  }
+
+  /**
+   * @param {number} a
+   * @param {number} b
+   * @param {number} c
+   * @param {number} d
+   * @return {RoundCornersAction}
+   */
+  radius(a:number, b?:number, c?:number, d?:number):this {
```

Comment:
Why not meaningful names?

Reply:
It's four identical radiuses, so..

radiusA, radiusB, radiusC, radiusD?

Reply:
As a user, how do I know how each one will affects the image? shouldn't their names and docstring convey that?

Reply:
Added the PHP Docstrings and links to the reference